contracts/UniswapV3Staker.sol

```diff
@@ -46,6 +46,9 @@ contract UniswapV3Staker is IUniswapV3Staker, Multicall {
         public immutable
         override nonfungiblePositionManager;
 
+    uint256 immutable maxDuration;
+    uint256 immutable maxTimeUntilStart;
```

Comment:
would it be more human readable as `maxIncentiveDurationSeconds` and `maxIncentiveStartLeadTime`?

Reply:
sure